src/app/app.module.ts

```diff
@@ -147,6 +148,22 @@ export function configUpdaterFactory(x): any {
   return x[0];
 }
 
+export function getOptionsFactory(arlasAuthService: AuthentificationService): any {
+  const getOptions = () => {
```

Comment:
No sure you need the getOptions variable, maybe this is expected (you can also `return () => {...`

Reply:
The getOptions variable is needed because of the compiler. 
He doesn't support returning a lambda function : 
https://angular.io/guide/aot-compiler#no-arrow-functions

The compiler is ok with assigning the lambda function to a variable and return it.